### ISSUE TITLE
Accessibility configurations for Video is missing "Others" tab

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#28]
## Expected Behavior

Should have the "Others" tab
![image](https://user-images.githubusercontent.com/993369/76173207-21da1200-617c-11ea-8903-a26df3c48ebd.png)

![image](https://user-images.githubusercontent.com/993369/76173211-2e5e6a80-617c-11ea-8ecb-e9499cc00a01.png)

## Current Behavior

![image](https://user-images.githubusercontent.com/993369/76173225-4e8e2980-617c-11ea-8c5d-8cc44f0f5a53.png)

## Possible Implementation

![image](https://user-images.githubusercontent.com/993369/76173489-b5144700-617e-11ea-872d-057c0f6f58e9.png)

See previous code 
[Line 140](https://gitlab.com/ecureuill/seli/-/blob/master/SELI-Platform/imports/components/tools/VerticalTab.js#L140)  and [Line 306](https://gitlab.com/ecureuill/seli/-/blob/master/SELI-Platform/imports/components/tools/VerticalTab.js#L306)

**Comment by CAndresH** [comment]
"Others" tab is enabled now.
![image](https://user-images.githubusercontent.com/50594090/77088992-3d8eb500-69d3-11ea-972a-0a7763eb951a.png)
